### DISCUSSION TITLE
ERGO and blocks api

### DISCUSSION THREAD

**Discussion by babelfish2001** [coinfoundry/miningcore#1279] [Q&A]
Hello, I have an ergo node setup and I setup Miningcore on the same computer.  

I am mining to it with other mining rigs on the same network.  I found three blocks and the API shows them. It appears the API has stopped updating the confirmation progress of the blocks.  Is this indicative of a larger problem or is this just an issue with the api?  I haven't found any additional blocks after the "http://127.0.0.1:4000/api/pool/ergo/blocks" page stopped responding and I am hopping that this issue isn't preventing me from finding additional blocks.

By not responding, I mean that I can still pull the page up but the data is not changing.

Everything else appears to be functioning....

**Comment by MiningCryptoLive** [comment]
Did you figure this out? Do the blocks show on  ergo explorer?

**Comment by babelfish2001** [comment]
They do show up on the explorer, it's just the API that stops updating.

**Reply by MiningCryptoLive** [reply]
That's good. Then something is not setup correctly for the API.

**Reply by oliverw** [reply]
> They do show up on the explorer, it's just the API that stops updating.

Not responding at all? No matter how often you retry?

**Reply by babelfish2001** [reply]
![image](https://user-images.githubusercontent.com/106462617/175043927-938af427-9606-4742-a085-b890a4d04e59.png)

Here is the blocks page from the api.  It just stayed looking like this even after the blocks were paid and in the wallet.

**Reply by oliverw** [reply]
Sounds like a caching issue to me. Ctrl+F5?

**Reply by babelfish2001** [reply]
I don't think so, but maybe?  Unfortunately, I uninstalled miningcore and postgre and then started over.  So that page is blank now...  Is there anyway to tell for sure if it is mining properly?

**Reply by babelfish2001** [reply]
And what would you recommend for the ergo.json file if I just want it to mine and payout to the node address?  Thank you for your time, I really appreciate it.
